packages/zmarkdown/plugins/client/zhtml.js

```diff
@@ -0,0 +1,95 @@
+const visit = require('unist-util-visit')
+
+const shortid = require('shortid')
+const clone = require('clone')
+const createWrapper = require('../../utils/wrappers')
+
+const remarkConfig = require('../../config/remark')
+
+const remark2rehype = require('remark-rehype')
+
+const rehypeAutolinkHeadings = require('rehype-autolink-headings')
+const rehypeFootnotesTitles = require('rehype-footnotes-title')
+const rehypeHighlight = require('rehype-highlight')
+const rehypeHTMLBlocks = require('rehype-html-blocks')
+const rehypeKatex = require('rehype-katex')
+const rehypeLineNumbers = require('../../utils/rehype-line-numbers')
+const rehypePostfixFootnotes = require('rehype-postfix-footnote-anchors')
+const rehypeSlug = require('rehype-slug')
+
+const rehypeStringify = require('rehype-stringify')
+
+const jsFiddleAndInaFilter = node => {
```

Comment:
Most of this file seems duplicated from the other one. What would you think of factoring the common parts out in a file that would define all of these utilities and exports them separately? (This file would also replace `zmdParser.js`)

```js
// …requires
// …definitions
module.exports = {
  foo,
  bar,
  baz
}
```

Reply:
To me this would be a structural change, not a minor cleanup. Would you do it that way or what are your thoughts?

Reply:
I have a little idea of ​​how to do that. I will do that tomorrow.